### PR TITLE
Added windows system warning

### PR DESCRIPTION
**Pull request by youngee91**
Added in new alerts for windows system #5

### CODE REVIEW

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml [hunk 1]
```diff
@@ -97,4 +97,51 @@ groups:
       summary: "Disk usage on / >70%"
       message: "{{ $labels.instance }}: / disk usage >70%. Current usage is {{ humanize $value }}."
 
+# CRITICAL alerts for windows server
+- name: windows_server_critical
+  rules:
+  - alert: windows_server_high_cpu_load_critical
+    expr: 100 - (avg by (instance) (rate(wmi_cpu_time_total{mode="idle"}[2m])) * 100) > 80
```
Comment (eugene-chow) [inline Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: Why is `rate == 2m`? Are you aware of the [difference between `irate` and `rate`](https://utcc.utoronto.ca/~cks/space/blog/sysadmin/PrometheusRateVsIrate)?

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml [hunk 2]
```diff
@@ -97,4 +97,51 @@ groups:
...
+    expr: 100 - (avg by (instance) (rate(wmi_cpu_time_total{mode="idle"}[2m])) * 100) > 8
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Windows Server CPU load (instance {{ $labels.instance }})"
+      message: "CPU Usage is more than 80% for 10 minutes\n  VALUE = {{ humanize $value }}\n  LABELS: {{ $labels }}"
+
+  - alert: windows_server_high_memory_load_warning
+    expr: 100 - 100 * wmi_os_physical_memory_free_bytes / wmi_cs_physical_memory_bytes > 8
```
Comment (eugene-chow) [inline Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: Can you confirm if `wmi_os_physical_memory_free_bytes` is the total free memory? In Linux, the physical free memory `node_memory_MemFree_bytes` doesn't include the free buffers and cache memory which should be part of the "free memory available".

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml [hunk 3]
```diff
@@ -97,4 +97,51 @@ groups:
...
+    for: 10m
+    labels:
+      severity: warning    annotations:      summary: "Windows Server memory load (instance {{ $labels.instance }})"
+      message: "Memory usage is more than 80% for 10 minutes\n  VALUE = {{ humanize $value }}\n  LABELS: {{ $labels }}"
+      
+  - alert: windows_server_disk_storage_warning
+    expr: 100.0 - 100 * ((wmi_logical_disk_free_bytes{} / 1024 / 1024 ) / (wmi_logical_disk_size_bytes{} / 1024 / 1024)) > 10
```
Comment (eugene-chow) [inline Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: Looks weird. Check the math and simplify it.

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml [hunk 4]
```diff
@@ -97,4 +97,51 @@ groups:
...
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Windows Server disk storage (instance {{ $labels.instance }})"
+      message: "Disk storage is more than 80%\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
```
Comment (eugene-chow) [inline Monitoring in a Box/mib/data/prometheus/config/alerts.yml:147]: Remove the newline. The alerts to Slack and other platforms will get truncated.

**Review by eugene-chow — CHANGES_REQUESTED** [review]
Did you test the Windows alerts?

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml (hunk 1 above)
Comment (youngee91) [reply Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: I have updated on the a file. Understood that irate takes the last two points within its range, which is instant.

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml (hunk 2 above)
Comment (youngee91) [reply Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: Based on what i saw from the [WMI exporter](https://github.com/martinlindhe/wmi_exporter/blob/086506121000ee0b7cf5e5100e46a0e79d570512/docs/collector.os.md), It describes as `Bytes of physical memory currently unused and available`

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml (hunk 3 above)
Comment (youngee91) [reply Monitoring in a Box/mib/data/prometheus/config/alerts.yml]: Updated on the latest file

File: Monitoring in a Box/mib/data/prometheus/config/alerts.yml (hunk 4 above)
Comment (youngee91) [reply Monitoring in a Box/mib/data/prometheus/config/alerts.yml:147]: Checked on Slack, newline is not truncated.

**Comment by youngee91** [comment]
Yes, i did test the windows alerts. Apologies for some of the errors as i forgot to update the `exp` after testing.

**Review by eugene-chow — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**